### PR TITLE
[Various] Trying to fix accidental reversion

### PR DESCRIPTION
**Pull request by kodster28**
As part of #3648, I did a bad production merge that accidentally reverted some of the other docs work done by other writers, as well as a few things done by the dev advocates.

Known issues:
- https://github.com/cloudflare/cloudflare-docs/commit/e80a1b6b673d0dfd767c4858d78259a670ff69f2
- #3647 
- Probably one of @deadlypants1973's PRs based on the unanticipated changes to the Workers repo

@lukeed, would you or another dev advocate mind helping on the `assets` files? Looks like I accidentally removed a closing brace and a `()`... not sure if that's been fixed elsewhere or needs to be included?

### CODE REVIEW

**Comment by kodster28** [comment]
Looks like this is also restoring your `format` changes in #3651, @lukeed.

File: content/waf/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ Cloudflare Specials </td> <td>
 100519 </td> <td>
 Magento - CVE:CVE-2022-24086 </td> <td>
 N/A </td> <td>
-Block </td> </tr> 
+Block </td> </tr>
+
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md:28]:
```suggestion
Block </td> </tr>
```

**Comment by kodster28** [comment]
Signoff from @deadlypants1973 on the updated changes to workers docs
Signoff from @abelinkinbio on the re-implementing the tunnel guides

File: content/waf/change-log/scheduled-changes.md (hunk 1 above)
Comment (kodster28) [reply content/waf/change-log/scheduled-changes.md:28]: Oh, that's weird. Hovering over this it says, `This suggestion is invalid b/c no changes were made to the code`

Comment (pedrosousa) [reply content/waf/change-log/scheduled-changes.md:28]: Well, I wanted to suggest an empty line removal... I failed. :)

**Review by pedrosousa — APPROVED** [review]

**Review by lukeed — APPROVED** [review]
Looks good, changes to `/assets/*` here don't matter

**Pull request merged**